lib/ansible/modules/network/itential/iap_token.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2018, Itential <opensource@itential.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+"""
+This module provides the token for Itential Automation Platform
+"""
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = """
+---
+module: iap_token
+version_added: "2.8"
+author: "Itential (@cma0) <opensource@itential.com>"
+short_description: Get token for the Itential Automation Platform
+description:
+  - Checks the connection to IAP and retrieves a login token.
+options:
+  iap_port:
+    description:
+      - Provide the port number for the Itential Automation Platform
+    required: true
+    default: null
+
+  iap_fqdn:
+    description:
+      - Provide the fqdn or ip-address for the Itential Automation Platform
+    required: true
+    default: null
+
+  username:
+    description:
+      - Provide the username for the Itential Automation Platform
+    required: true
+    default: null
+
+  password:
+    description:
+      - Provide the password for the Itential Automation Platform
+    required: true
+    default: null
+
+  https:
+    description:
+      - Use HTTPS to connect
+      - By default using http
+    type: bool
+    default: False
+
+  validate_certs:
+    description:
+      - If C(no), SSL certificates for the target url will not be validated. This should only be used
+        on personally controlled sites using self-signed certificates.
+    type: bool
+    default: False
+
+notes:
+  - This module is under construction
```

Comment:
Specify what isn't implemented

Reply:
@gundalow Removed the remark. Thanks again.